plat/kvm/Config.uk

```diff
@@ -29,13 +29,13 @@ config KVM_BOOT_PROTO_LXBOOT
 	help
 		Linux 64-bit Boot Protocol
 
-config KVM_BOOT_PROTOCOL_QEMU_VIRT
```

Comment:
```
Rename the Kconfig options for QEMU virt and EFI stub to make them
with the rest of boot protocol options.
```
to
```
Rename the Kconfig options for QEMU virt and EFI stub to align them
with the rest of boot protocol options.
```
for commit `plat/kvm: Make Kconfig boot protocol option naming consistent`